packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -39,6 +39,7 @@ import {
   StarIcon,
   ConnectedSquareIcon,
   ClockWithArrowIcon,
+  ListWithSearch,
```

Comment:
This doesn't match the other icons imported from UI Kit, perhaps...
```suggestion
  ListWithSearchIcon,
```

Looking at UI Kit, it's quite possible the icon is not exported correctly 😔

Reply:
https://github.com/commercetools/ui-kit/pull/1791

Reply:
I'll bump the version again once that PR is merged! Thanks for the fix.